### DISCUSSION TITLE
Restrict Movement during 'CueTarget'

### DISCUSSION THREAD

**Discussion by SR-gituser** [mjstarrett/Landmarks#29] [Q&A]
Hello :) 
Any potential feedback on the following would be appreciated...

We're trying to disable both rotational and translational movement during the 'CueTarget' phase using 'Restrict Movement'. 
Is the 'Restrict Movement' checkbox associated with 'CueTarget' for: disabling translational movement (forward/back/left/right) whilst rotational movement remains enabled? 

Kind regards, SR-gituser

![Restrict_Movement](https://github.com/user-attachments/assets/15fcc5b2-f58b-4c77-bdaa-d257c8b8db23)

**Comment by mjstarrett** [comment]
In the current version, restrictMovement only disables translations (i.e., rotations remain intact). A yet-to-be-released update does expand this to include both translation and rotations.

If restricting both translations and rotations is something you are hoping to have currently, please open an issue/feature-request and I can try to push the relevant code from the update to a new branch. Please just keep in mind that this branch and associated changes will not have been fully tested with the current version of the `main` branch.

**Reply by SR-gituser** [reply]
Thank you for the feedback around this feature, a request was opened :)
